apps/web/lib/features/task/task-status.tsx

```diff
@@ -330,16 +338,21 @@ export function VersionPropertiesDropown({
 	forDetails,
 	multiple,
 	sidebarUI = false,
+	whiteBg = false,
```

Comment:
@desperado1802 Instead of using this flag `whiteBg`, let us update default color while creating Version.

You can check `version-form.tsx`.